### PR TITLE
Feat: easyPoints SDK integration files

### PR DESCRIPTION
**Pull request by Tw33t3r**

### CODE REVIEW

File: assets/easy_points.js
Comment (NexPB) [inline assets/easy_points.js:1]: We did not replace all of this, right? We can only delete parts here.

File: assets/easy_points_integration.js [hunk 1]
```diff
@@ -19,43 +8,7 @@
...
-  //     }
-  //   };
-
-  //   (new MutationObserver(callback))
-  //     .observe(cartNode, {
-  //       attributes: false,
-  //       childList: true,
-  //       subtree: true
-  //     });
-  // }
```
Comment (NexPB) [inline assets/easy_points_integration.js:58]: Please keep this as it is used as a reference

File: assets/easy_points_integration.js [hunk 2]
```diff
@@ -1,15 +1,4 @@
-/**
- * v1.91
- *
```
Comment (NexPB) [inline assets/easy_points_integration.js:3]: I'd like to keep this, and even bump it to v2.0

File: assets/easy_points_integration.js [hunk 3]
```diff
@@ -546,114 +499,41 @@ var EasyPoints = {
...
   },
 
-  Tiers: {
-    /**
-     * Recalculates the next tier rank and updates the associated HTML elements accordingly.
-     * This function also handles the case where the next tier is not present (maximum tier reached).
-     *
-     * @param {number|null} [subtotal=null] - The subtotal amount used to calculate the next tier. If null, the function will attempt to retrieve it from an HTML element.
-     */
-    recalculate: function(subtotal = null) {
```
Comment (NexPB) [inline assets/easy_points_integration.js:556]: This is deleted but still used

File: snippets/easy_points_redemption_form.liquid [hunk 4]
```diff
@@ -25,6 +25,9 @@
 
       <input type="hidden" id="redemption-max-points" name="coupon[max_redeemable]" value="{{ max_redeemable }}">
       <input type="{%- if hide_points_form -%}hidden{%- else -%}text{%- endif -%}" id="redemption-point-value" name="coupon[point_value]" value="0">
+      {%- if customer -%}
+        <input type="hidden" id="redemption-balance" name="balance" value="{{ customer.metafields.loyalty.easy_points_attributes.value.balance }}">
+      {%- endif -%}
```
Comment (NexPB) [inline snippets/easy_points_redemption_form.liquid:30]: Why was this added?

**Review by NexPB — CHANGES_REQUESTED** [review]

**Comment by wjlee2020** [comment]
just my two cents: it'd be cool if we could update this to typescript. and let's try to make this into a private repo so assholes like me don't make comments

**Review by NexPB — APPROVED** [review]

**Pull request merged**